platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCPeopleTestIT.java

```diff
@@ -0,0 +1,473 @@
+                          .parent()
+                          .parent()
+                          .parent()
+                          .find(byText(connect))
+                          .should(Condition.exist);
+    manageLogInOut.signIn(DATA_USER1, "gtngtn");
+    navigationToolbar.goToManageCommunity();
+    addUsers.deleteUser(username1);
+    addUsers.deleteUser(username2);
+    addUsers.deleteUser(username3);
+    addUsers.deleteUser(username4);
+    addUsers.deleteUser(username5);
+  }
+
+  /**
+   * <li>Case ID:121900.</li>
+   * <li>Test Case Name: Pending Requests list.</li>
+   * <li>Case ID:121901.</li>
+   * <li>Test Case Name: Received Invitations</li> Step Number: 1 Step Name: Step
+   * 1: Open Pending Requests list Step Description: - Login intranet home -
+   * Access people list, invite an user - Go to My Connections/Requests Sent -
+   * Click Cancel Request Input Data: Expected Outcome: - Display all user's
+   * requests - Requested user is removed from list
+   */
+  @Test
+  @BugInPLF("SOC-5738")
+  public void test05_PendingRequests() {
+    info("test05_PendingRequestsList");
+    String username1 = "usernamea" + getRandomString();
+    String email1 = username1 + "@test.com";
+    String username2 = "usernameb" + getRandomString();
+    String email2 = username2 + "@test.com";
+    String password = "123456";
+
+    info("Add user");
+    navigationToolbar.goToAddUser();
+    addUsers.addUser(username1, password, email1, username1, username1);
+    addUsers.addUser(username2, password, email2, username2, username2);
+    manageLogInOut.signIn(username1, password);
+
+    info("Click on Connections on the left panel");
+    homePagePlatform.goToConnections();
+    info("Click on Connect button to invite an user");
+    connectionsManagement.connectToAUser(username2);
+    $(ELEMENT_CONNECTION_CANCEL_BTN).waitUntil(Condition.appears, Configuration.timeout);
+    connectionsManagement.goToConnectionTab(ConnectionsManagement.selectTabOption.PENDING);
+    $(ELEMENT_CONNECTION_CANCEL_BTN).waitUntil(Condition.appears, Configuration.timeout);
+
+    info("Cancel request");
+    connectionsManagement.cancelConnection(username2);
+
+    info("test06_ReceivedInvitations");
+    homePagePlatform.goToConnections();
+    connectionsManagement.goToConnectionTab(ConnectionsManagement.selectTabOption.ALL);
+    connectionsManagement.connectToAUser(username2);
+    $(ELEMENT_CONNECTION_CANCEL_BTN).waitUntil(Condition.appears, Configuration.timeout);
+
+    manageLogInOut.signIn(username2, password);
+    homePagePlatform.goToConnections();
+    connectionsManagement.goToConnectionTab(ConnectionsManagement.selectTabOption.RECEIVE);
+    $(ELEMENT_CONNECTION_CONFIRM_BTN).waitUntil(Condition.appears, Configuration.timeout);
+    manageLogInOut.signIn(DATA_USER1, "gtngtn");
+    navigationToolbar.goToManageCommunity();
+    addUsers.deleteUser(username1);
+    addUsers.deleteUser(username2);
+
+  }
+
+  @Test
+  @BugInPLF("SOC-5738")
```

Comment:
Annotations should be set next to the test method.

Reply:
Done